schemas/com.snowplowanalytics.snowplow/application_background/jsonschema/1-0-0

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an application background event",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "application_background",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+
+  "type": "object",
+  "properties": {
+    "index": {
+      "type": "integer",
```

Comment:
Maybe just `eventIndex` then.  It states more that it an index to do with the wrapping event rather than anything else to do with fragments/activities etc.